sample/FOSSASIASummit2017/GoogleIO17/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@ParthS007 Incorrect path for GoogleIO2017. It should not be inside fossaissummit2017 directory

Reply:
@geekyd  I have not changed it, It is same as before

Reply:
Ohh, by mistake.